electron_strings.grdp

```diff
@@ -79,5 +79,9 @@
     <message name="IDS_UTILITY_PROCESS_UTILITY_WIN_NAME" desc="The name of the utility process used to handle Windows utility operations.">
       Windows Utilities
     </message>
+    <message name="IDS_TOAST_NOTIFICATION_DISMISS_TEXT" desc="Text label for the Dismiss button in Windows Toast Notifications. This is used when the toast has a timeout type of never.">
```

Comment:
I don't know if we need to create a new string? Upstream already has 

```
      <message name="IDS_APP_CLOSE" desc="A generic term for Close on buttons and menus.">
        Close
      </message>
```

in `ui/strings/`

Reply:
We also have https://github.com/electron/electron/blob/8bbdc224ac3d7756af930ce946a59ce5ff2533b5/electron_strings.grdp#L54-L56 - I think Dismiss is slightly more in line with what [Toast Content docs](https://docs.microsoft.com/en-us/windows/uwp/design/shell/tiles-and-notifications/adaptive-interactive-toasts#snoozedismiss) have but I don't feel strongly

Reply:
It just seems like pointless translation work & lack-of-translations when Chromium already has a fully-translated string sitting on the shelf :smiley: 

OTOH if we want to use "Dismiss" rather than "Close" for desktop consistency, we still may be able to use existing strings. That link says:

> Optionally specify a custom content string:
>
> If you don't provide a string, we'll automatically use localized strings for "Snooze" and "Dismiss".

It sounds like they're saying we can use `<action activationType="system" arguments="dismiss" content=""/>` and Windows will fill in the default string?

Reply:
Heh unfortunately that doc is a bit misleading - @mlaurencin figured out yesterday that that only applies to the _Windows_ API and not the generated XML - if `content` is left blank in our code (e.g. we don't supply a string) then the button simply does not show up 🙃

Reply:
OK. In that case I'd vote for reusing the Chromium string instead of reusing the nonexistent Windows string :smile:

Reply:
sound good to me! @mlaurencin would you mind updating that?

Reply:
Got it!